src/keylessBackup/KeylessBackupProgress.tsx

```diff
@@ -89,7 +101,41 @@ function Restore() {
         </SafeAreaView>
       )
     }
-    // TODO(ACT-781): Implement Success screen
+    case KeylessBackupStatus.Completed: {
+      return (
+        <SafeAreaView style={styles.container}>
+          <View style={styles.finishedContainer}>
+            <View style={styles.iconContainer}>
+              <GreenLoadingSpinnerToCheck />
+            </View>
+            <Text style={styles.title}>{t('keylessBackupStatus.restore.completed.title')}</Text>
+            <Text style={styles.body}>
+              {totalBalanceLocal.gt(0)
+                ? t('keylessBackupStatus.restore.completed.bodyBalance', {
+                    localCurrencySymbol,
+                    totalBalance: totalBalanceLocal.toFormat(2),
+                  })
+                : t('keylessBackupStatus.restore.completed.bodyZeroBalance')}
```

Comment:
I know the only difference on this screen is the copy if the balance is 0, but maybe this is something worth testing (showing the 0 balance copy vs. normal copy) since it relies on a handful of logic to display.